dataactbroker/permissions.py

```diff
@@ -39,7 +39,7 @@ def decorated_function(*args, **kwargs):
                                 break
 
                     finally:
-                        InterfaceHolder.closeOne(userDb)
+                        userDb.close()
```

Comment:
Does this mean we're destroying the userdb engine every time this decorator is used? Could that be a contributing factor to the multitude of db connections that @kevinli-work and @klrBAH were discussing yesterday?

I get that this PR doesn't change the behavior and is just executing it a different way. Could well be something for me to look at when I'm doing the db story, but am curious if you know anything about the original intent.